### DISCUSSION TITLE
Helm chart for deployment

### DISCUSSION THREAD

**Discussion by sunchao** [unitycatalog/unitycatalog#276] [Ideas]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you would like**

For people who are running in Kubernetes, it would be very convenient if the project can offer a official Helm chart so we can quickly try it out in the environment.

**Additional context**

**Comment by MrPowers** [comment]
Thanks for raising this @sunchao - going to convert this to a discussion, so the community can chime in on this wonderful idea!

**Comment by MrPowers** [comment]
I just realized that there is an open PR for this @sunchao: https://github.com/unitycatalog/unitycatalog/pull/45

Can you take a look at the PR and let me know what you think?

**Reply by dnskr** [reply]
Suggested implementation in the PR looks too basic and does not take into account the specifics of Unity Catalog application. Also, it seems that it was generated with an out-of-date Helm cli tool because it requires Kubernetes 1.12+. Usually only latest three versions of Kubernetes are supported (1.30, 1.29, 1.28 at the moment) to avoid complex templating for support of different API versions like https://github.com/unitycatalog/unitycatalog/blob/6f36f362ff58ebb3226527de2d4f8c1e2c7775c9/helm/unitycatalog/templates/ingress.yaml#L9-L14

**Reply by dennyglee** [reply]
Thanks @dnskr - by any chance could you also provide your feedback to the PR itself so we can work on either updating it and/or create a new version of it if/when appropriate?
